infrastructure/tooling/src/importer/importer.js

```diff
@@ -25,14 +26,37 @@ const importer = async options => {
       provides: [tableName],
       run: async (requirements, utils) => {
         const start = new Date();
-        const entities = await readAzureTable({azureCreds: credentials.azure, tableName, utils});
 
-        await writeToPostgres(tableName, entities, db, ALLOWED_TABLES);
+        const pgTable = postgresTableName(tableName);
+
+        await db._withClient('admin', async client => {
+          await client.query(`truncate ${pgTable}`);
+        });
+
+        let rowsImported = 0;
+        async function importTable(tableParameters = {}, rowsProcessed = 0) {
+          const { entities, tableParams, count } = await readAzureTable({
+            azureCreds: credentials.azure,
+            tableName,
+            utils,
+            tableParams: tableParameters,
+            rowsProcessed,
+          });
+          await writeToPostgres(tableName, entities, db, ALLOWED_TABLES);
+
+          if (tableParams.nextPartitionKey && tableParams.nextRowKey) {
+            await importTable(tableParams, count);
```

Comment:
I wonder if this kind of recursion will end up keeping `entities` around on the call stack until the entire process is done.  I don't know how smart Node's optimization is.  Could this instead `return importTable(tableParams, count)`?

Reply:
Good observation. Done!